index.html

```diff
@@ -46,5 +47,178 @@ <h1>I’m Rashmi.<br> to see<br> you!</h1>
+                            </ul>
+                        </div>
+                        <div >
+                             <button class="button-A">See project</button>
+                        </div>
+                    </div>
+                
+      </div>
+                <div class="grid-item"><img src="img/Snapshoot Portfolio (1).png">
+                    <div class="detail">
+                        <h1>Tonic</h1>
+                        <div class="list-01">
+                            <ul>
+                                <li class="canopy">CANOPY</li>
+                                <li><img src="img/Counter.png"></li>
+                                <li>Back End Dev</li>
+                                <li><img src="img/Counter.png"></li>
+                                <li>2015</li>
+                            </ul>
+                        </div>
+                        <p>A daily selection of privately<br> 
+                            personalized reads; no accounts or<br> 
+                            sign-ups required.
+                        </p>
+                        <div class="lang">
+                            <ul>
+                                <li>HTML</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                        <div >
+                             <button class="button-A">See project</button>
+                        </div>
+                    </div>
+      </div>
+                <div class="grid-item"><img src="img/Snapshoot Portfolio (3).png">
+                    <div class="detail">
+                        <h1>Tonic</h1>
+                        <div class="list-01">
+                            <ul>
+                                <li class="canopy">CANOPY</li>
+                                <li><img src="img/Counter.png"></li>
+                                <li>Back End Dev</li>
+                                <li><img src="img/Counter.png"></li>
+                                <li>2015</li>
+                            </ul>
+                        </div>
+                        <p>A daily selection of privately<br> 
+                            personalized reads; no accounts or<br> 
+                            sign-ups required.
+                        </p>
+                        <div class="lang">
+                            <ul>
+                                <li>HTML</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                        <div >
+                             <button class="button-A">See project</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        
+        <section class="about-myself">
```

Comment:
According to the Figma design, I suggest you stick to the design as much as possible.

As I mentioned above, please, make sure you use the right:

- Margin & padding
- font
- and color

> ## The Figma Design
>
><img width="228" alt="Screenshot 2023-06-03 at 6 23 02 PM" src="https://github.com/siriwardhanage/My-first-portfolio-/assets/91237525/14acc904-577a-4b25-92cc-9f2872eb92c7">
>
> ## Your Design
>
><img width="372" alt="Screenshot 2023-06-03 at 6 23 31 PM" src="https://github.com/siriwardhanage/My-first-portfolio-/assets/91237525/e6ed288b-616a-4870-a472-a636ca720f4c">